ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/backwardsync/ForwardSyncStep.java

```diff
@@ -95,6 +147,25 @@ protected BlockHeader processKnownAncestors(final Void unused) {
     return run.thenApply(AbstractPeerTask.PeerTaskResult::getResult);
   }
 
+  @VisibleForTesting
+  protected CompletableFuture<List<Block>> requestBodies(final List<BlockHeader> blockHeaders) {
+    final GetBodiesFromPeerTask getBodiesFromPeerTask =
```

Comment:
Maybe we need to use the retry implementation here ?

Reply:
I think I will create a ticket to resolve this later.

Reply:
I raised an [issue 3613](https://github.com/hyperledger/besu/issues/3613)